client/tailwind.js

```diff
@@ -10,118 +10,74 @@ module.exports = {
     colors: {
       transparent: 'transparent',
 
-      black: '#000',
-      white: '#fff',
-
-      gray: {
-        100: '#f7fafc',
-        200: '#edf2f7',
-        300: '#e2e8f0',
-        400: '#cbd5e0',
-        500: '#a0aec0',
-        600: '#718096',
-        700: '#4a5568',
-        800: '#2d3748',
-        900: '#1a202c',
+      black: '#16171A',
+      white: '#FFFFFF',
+      navy: '#0F015E',
+      primary: {
+        default: '#4400CC',
+        100: '#E8E5FF',
+        200: '#DDD9FF',
+        300: '#5856D6',
+        400: '#7B16FF',
+        500: '#2A0080',
+      },
```

Comment:
The `primary` ref should only be one. In this case I think it should be `#7B16FF`.

```suggestion
primary: '#7B16FF',
purple: {
    100: '#E8E5FF',
    200: '#DDD9FF',
    300: '#5856D6',
    400: '#7B16FF',
    500: '#2A0080'
}
```